ngraph/frontend/onnx_import/src/op/reduce.cpp

```diff
@@ -20,131 +20,132 @@
 #include "ngraph/builder/norm.hpp"
 #include "ngraph/node.hpp"
 #include "onnx_import/default_opset.hpp"
-#include "onnx_import/utils/reduction.hpp"
-#include "reduce.hpp"
+#include "onnx_import/exceptions.hpp"
+#include "onnx_import/utils/common.hpp"
 
 namespace ngraph
 {
     namespace onnx_import
     {
         namespace op
         {
+            namespace
+            {
+                std::shared_ptr<default_opset::Constant> get_reduction_axes(const Node& node)
+                {
+                    auto reduction_axes =
```

Comment:
minor: I think that more clear is  using `node.has_attribute` instead of checking `reduction_axes.empty()`

Reply:
With the latest [ONNX ReduceSum](https://github.com/onnx/onnx/blob/master/docs/Operators.md#reducesum), `axes` is no longer attribute but input. There is other additional attribute `noop_with_empty_axes` to define such a case, so no axes and empty axes can be not equivalent:
> Defines behaviour if 'axes' is empty. Default behaviour with 'false' is to reduce all axes. When axes is empty and this attribute is set to true, input tensor will not be reduced, and the output tensor would be equivalent to input tensor.

Reply:
This is a good spot but it's a missing support of the newest version of this operator. It's out of scope of this PR.

Reply:
Ok, but with the change 9da1f16 we no longer replace empty `axes` vector by default, also for previous versions. Is it correct?

Reply:
Right, I've reverted this commit not to change the current behavior. I've also created a ticket for the missing implementation of ReduceSum@13